### PR TITLE
Add CompiledModel.inputs property

### PR DESCRIPTION
**Pull request by almilosz**
### Details:
- Expose CompiledModel.inputs property
- Expose CompiledModel.input() method
- Expose CompiledModel.output() method
- Expose Input class

### Tickets
- 114040
- 114041

### CODE REVIEW

File: src/bindings/js/node/src/compiled_model.cpp [hunk 1]
```diff
@@ -46,3 +73,37 @@ Napi::Value CompiledModelWrap::get_outputs(const Napi::CallbackInfo& info) {
...
+        }
+    } else if (info.Length() != 1) {
+        reportError(info.Env(), "Invalid number of arguments -> " + std::to_string(info.Length()));
+        return Napi::Value();
+    } else if (info[0].IsString()) {
+        auto tensor_name = info[0].ToString();
+        return Output::Wrap(info.Env(), _compiled_model.input(tensor_name));
+    } else if (info[0].IsNumber()) {
+        auto idx = info[0].As<Napi::Number>().Int32Value();
+        return Output::Wrap(info.Env(), _compiled_model.input(idx));
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/src/compiled_model.cpp:93]: If passed not existed index, error appears.
Do we need to catch this error or original looks good on js side?

File: src/bindings/js/node/src/node_output.cpp [hunk 2]
```diff
@@ -32,7 +28,7 @@ Napi::Object Output::Wrap(Napi::Env env, ov::Output<const ov::Node> output) {
     Napi::HandleScope scope(env);
     Napi::Object obj = GetClassConstructor(env).New({});
     Output* output_ptr = Napi::ObjectWrap<Output>::Unwrap(obj);
-    output_ptr->set_output(output);
+    output_ptr->_output = output;
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/src/node_output.cpp:31]: **_output** is a private property. Does this work?

**Review by vishniakov-nikolai — APPROVED** [review]
Looks good, but I am not sure about assignment to private property.

File: src/bindings/js/node/src/compiled_model.cpp (hunk 1 above)
Comment (almilosz) [reply src/bindings/js/node/src/compiled_model.cpp:93]: Original error from C++ says: 

> Check 'i < _impl->outputs().size()' failed at src/inference/src/compiled_model.cpp:86:
Cannot get output for index: 6 outputs size is 1

I think it is clear where the error comes from, but when we add tests on js side (e.g using JEST) we will have to report this error using Napi::Error which is a representation of the JavaScript Error object.

**Pull request merged**